cpp/src/arrow/compute/kernels/scalar_string.cc

```diff
@@ -15,13 +15,15 @@
 // specific language governing permissions and limitations
 // under the License.
 
+#include <utf8proc.h>
```

Comment:
Nit, but please separate this include from the standard library includes (and put it below them).

Reply:
Np, I'd rather do it correct.